js/ui/overlay.js

```diff
@@ -559,7 +559,8 @@ var Overlay = Widget.inherit({
         }
 
         var $container = this._$content,
-            outsideClick = !($container.is(e.target) || domUtils.contains($container.get(0), e.target));
+            isDetachedTarget = !$(e.target).closest(window.document).length,
```

Comment:
Can we use `domUtils.contains(window.document, e.target)` instead?

Reply:
No, we can not. When the `e.target` is the `window.document`, `contains` method will return `false` because it will search document inside of the body but `closest` will return `true`

Reply:
Ok

Reply:
BTW:

```
...
isConnectedTarget = $(target).is(document) || domUtils.contains(document, target),
outsideClick = isConnectedTarget && !($container.is(e.target) || domUtils.contains($container.get(0), e.target));
```